scanpy/queries/_queries.py

```diff
@@ -93,7 +93,8 @@ def biomart_annotations(
 
     Returns
     -------
-    A `pd.DataFrame` containing annotations.
+    :class:`pandas.DataFrame`
+        Dataframe containing annotations.
```

Comment:
Due to recent doc changes: If you just have one return value, write prose and use a return value annotation.

(same for every other time you changed this in this commit)

Reply:
Just changed.

Also, I think it would be reasonable to include this with `v1.5`, if `v1.4.1` is otherwise just bug fixes.

Reply:
sure!